### PR TITLE
Little test refactoring

### PR DESCRIPTION
**Pull request by MathisGD**

### CODE REVIEW

File: test/forge/ERC20WrapperBundlerLocalTest.sol [hunk 1]
```diff
@@ -3,11 +3,12 @@ pragma solidity ^0.8.0;
 
 import {ErrorsLib} from "../../src/libraries/ErrorsLib.sol";
 
-import {ERC20WrapperBundlerMock} from "../../src/mocks/bundlers/ERC20WrapperBundlerMock.sol";
 import {ERC20WrapperMock, ERC20Wrapper} from "../../src/mocks/ERC20WrapperMock.sol";
 
 import "./helpers/LocalTest.sol";
 
+contract ERC20WrapperBundlerMock is ERC20WrapperBundler, TransferBundler {}
```
Comment (QGarchery) [inline test/forge/ERC20WrapperBundlerLocalTest.sol:10]: I thought you were going to remove mock bundlers entirely and only test the full bundler. What made you change your mind ?

Comment (MathisGD) [reply test/forge/ERC20WrapperBundlerLocalTest.sol:10]: I thought to do this too. But these tests are chain agnostic, so what bundler should we even be testing ?

**Review by MerlinEgalite — APPROVED** [review]

File: test/forge/ERC20WrapperBundlerLocalTest.sol (hunk 1 above)
Comment (QGarchery) [reply test/forge/ERC20WrapperBundlerLocalTest.sol:10]: Maybe one big mock bundler with all the chain agnostic sub-bundlers ? That way we don't define 2 contracts in one file

File: test/forge/PermitBundlerLocalTest.sol [hunk 2]
```diff
@@ -3,11 +3,14 @@ pragma solidity ^0.8.0;
 
 import {SigUtils, Permit} from "./helpers/SigUtils.sol";
 
-import "../../src/mocks/bundlers/PermitBundlerMock.sol";
+import {ErrorsLib} from "src/libraries/ErrorsLib.sol";
+import {IERC20Permit} from "openzeppelin-contracts/contracts/token/ERC20/extensions/IERC20Permit.sol";
```
Comment (Rubilmax) [inline test/forge/PermitBundlerLocalTest.sol:7]: These don't use relative paths, which can be useful if we want to extend these tests in a repository using this repository as a submodule :D

Comment (MathisGD) [reply test/forge/PermitBundlerLocalTest.sol:7]: nice catch, will fix

**Review by Rubilmax — APPROVED** [review]

**Comment by MerlinEgalite** [comment]
some conflicts cc @MathisGD

**Review by QGarchery — APPROVED** [review]

**Pull request merged**